packages/backend/src/modules/task/task.service.ts

```diff
@@ -28,6 +28,10 @@ export class TaskService {
   findById(_id: string): Promise<Task> {
     return this._taskRepository.findById(_id)
   }
+
+  findByIdAndRemove(_id: string): Promise<Task> {
```

Comment:
what id? taskId or dashboardId? It would be good to use both

Reply:
ofc task id